ihontaryk/homework_11/guess_word.py

```diff
@@ -0,0 +1,129 @@
+import random
+import csv
+from enum import Enum
+from pathlib import Path
+import re
+
+
+class CsvFileReader:
+    """
+    Class for reading and modifying CSV file
+    """
+
+    def __init__(self, csv_file, directory="homework_11"):
+        """
+        Constructor for CSV file
+        """
+        self.csv_file = csv_file
+        self.directory = directory
+        base_dir = Path(__file__).resolve().parent.parent
+        self.data_file = base_dir.joinpath(directory).joinpath(csv_file)
+        self.data = self.get_data
+
+    @property
+    def get_data(self):
```

Comment:
since it is verb it should not be a property

and I am not sure if you need class for this action, simple function: get_random_line  should be enough

Reply:
if you want CsvReader, what does it actually do for you?